public/style/components/_data.scss

```diff
@@ -0,0 +1,10 @@
+.data-primary {
```

Comment:
If data is purely related to forecast, should it be something like `.forecast-data` to show the relation? Could then move into `_forecast.scss`